src/core/modules/Planet.cpp

```diff
@@ -2620,6 +2620,11 @@ void Planet::draw3dModel(StelCore* core, StelProjector::ModelViewTranformP trans
 		transfo2->combine(Mat4d::zrotation(M_PI_180*static_cast<double>(axisRotation + 90.f)));
 		StelPainter sPainter(core->getProjection(transfo2));
 		gl = sPainter.glFuncs();
+
+        if(StelApp::getInstance().getSettings()->value("video/multisampling", 0).toUInt() != 0)
```

Comment:
I think this part should be moved into init() and here just use some variable to speed-up comparison and avoid extra reading config.ini data

Reply:
Moved to the constructor